ui/src/index.ts

```diff
@@ -71,9 +71,11 @@ window.addEventListener("load", async function () {
           strokeMiterlimit="10"
         />
       </svg>
-    </div>
-    Hold on, we're retrieving the data you requested ...
-  </h2>`;
+    </div>`;
+
+  const statsLoader = `<h2>${htmlLoader} Hold on, we're retrieving the data you requested ...</h2>`;
```

Comment:
For accessibility, can we use something else than a h2 to display information that is not really a title ?

Reply:
I put a div instead